cli/__tests__/logger.test.js

```diff
@@ -9,21 +9,28 @@ describe('logger', () => {
   const oneArgMethods = [
     { method: 'log', expectedPrefix: '' },
     { method: 'task', expectedPrefix: '👍  ' },
-    { method: 'start', expectedPrefix: '🔥  ' },
-    { method: 'end', expectedPrefix: '✅  ' },
+    { method: 'start', expectedPrefix: '\n🔥  ' },
+    { method: 'end', expectedPrefix: '\n✅  ' },
     { method: 'info', expectedPrefix: 'ℹ️  ' },
   ];
 
   const twoArgMethods = [
     { method: 'warn', expectedPrefix: '🙀  ' },
-    { method: 'error', expectedPrefix: '❌  ' },
+    { method: 'error', expectedPrefix: '\n❌  ' },
     { method: 'debug', expectedPrefix: '🐞  ' },
   ];
 
+  const newlineMethods = ['start', 'end', 'error'];
+
   [...oneArgMethods, ...twoArgMethods].forEach(({ method, expectedPrefix }) => {
     it(`logger.${method}: simple usage`, () => {
       logger[method]('here is some text');
-      expect(global.console.log.mock.calls).toEqual([[`${expectedPrefix}here is some text`]]);
+      if (newlineMethods.indexOf(method) > -1) {
+        expect(global.console.log.mock.calls)
+          .toEqual([[`${expectedPrefix}here is some text`], []]);
+      } else {
+        expect(global.console.log.mock.calls).toEqual([[`${expectedPrefix}here is some text`]]);
```

Comment:
Yea, I think it's a bit too opaque, I think something like this is more descriptive:

```js
expect(global.console.log).toBeCalledWith(`${expectedPrefix}here is some text`);
expect(global.console.log.mock.calls).toEqual(1);
```

Or, when we update to `Jest v16`:

```js
expect(global.console.log).toBeCalledWith(`${expectedPrefix}here is some text`);
expect(global.console.log).toHaveBeenCalledTimes(1);
```